src/codegens/golang.ts

```diff
@@ -133,9 +135,49 @@ export default class Golang extends CodeGen {
       return [...typings, `\t${title.padEnd(titleMaxLength)} *${title}`];
     }, []);
 
+    const tit = this.getSafeTitle(s.title as string);
+
+    // Assuming that AnyOfs are either an {}, or an [];
+    // ie. there are TWO possibilities of reduction;
+    // either a single thing or an array (slice) of things.
+    // ALSO
+    // Assuming that the first title represents the single,
+    // and the second represents the set (arraytype).
+
+    const anyOfOneTit = titles[0];
+    const anyOfTwoTit = titles[1];
```

Comment:
This needs to be fixed so that we're sure we're getting the title of the object instead of the array.